frontend/pages/campaigns/index.tsx

```diff
@@ -32,13 +37,27 @@ const sampleCampaign: CampaignListData = {
 const sampleCampaigns: CampaignListData[] = Array(11).fill(sampleCampaign);
 
 const Campaigns = () => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+  const [searchDrawerIsOpen, setSearchDrawerIsOpen] = useState<boolean>(false);
+
   return (
     <Box>
       <Head>
         <title>Campaigns</title>
       </Head>
 
-      <CampaignList campaigns={sampleCampaigns} />
+      <Container sx={containerSx} component="main">
+        <CampaignSearch searchDrawerIsOpen={searchDrawerIsOpen} setSearchDrawerIsOpen={setSearchDrawerIsOpen} />
+
+        <CampaignList campaigns={sampleCampaigns} />
+
+        {isMobile && (
+          <Fab sx={mobileSearchButtonSx} onClick={() => setSearchDrawerIsOpen(true)}>
+            <SearchIcon fontSize="large" />
+          </Fab>
```

Comment:
Do you think it's better to have the search toggle in the top right?

Most other apps have search at the top and not at the bottom.

Reply:
Also if we are moving to the top, a normal search icon will do (without the yellow bg).

Reply:
KIV this first as it's not an easy change.